apps/console/src/features/email-templates/components/email-template-editor.tsx

```diff
@@ -96,10 +113,80 @@ export const EmailTemplateEditor: FunctionComponent<EmailTemplateEditorPropsInte
              */
             iframeDoc.open();
             iframeDoc.close();
-            iframeDoc.body.innerHTML = updatedContent.current;
+            iframeDoc.body.innerHTML = replaceBrandingPlaceholders(updatedContent.current, brandingPreferences);
         }
     };
 
+    /**
+     * Replace branding related placeholders in email templates.
+     *
+     * @param   templateContent    - template content
+     * @param   brandingPreference - branding preference object
+     * @returns preview content
+     */
+    const replaceBrandingPlaceholders = (
+        templateContent: string,
+        brandingPreference: BrandingPreference
+    ): string => {
+
+        const isBrandingEnabled:boolean = (brandingPreference != null && brandingPreference.configs.isBrandingEnabled);
+
+        let theme:string;
```

Comment:
Lets assign the default values first and get rid of the else block.

```suggestion
        let theme:string = EmailTemplateManagementConstants.DEFAULT_BRANDING_ACTIVE_THEME;
```